### ISSUE TITLE
Home panel

### ISSUE THREAD

**Issue by bdaase** [xou816/spot#468]
It would be nice to have the Spotify `Home` panel implemented in Spot.

It is quite nice to discover new content. Additionally, there is otherwise no way to get the `Discover Weekly` playlist.

**Comment by artemcodes** [comment]
There is a way
https://www.spotify.com/us/download/linux/

**Comment by jannuary** [comment]
I'd be careful of exposing algorithmic recommendations so prominently - in contrast to Spotify, we focus more on the user's library.

Though I suppose we could have a 'For You' view that has mixes and such

**Comment by laurinneff** [comment]
> Additionally, there is otherwise no way to get the Discover Weekly playlist.

You can add it to your library (heart button) in one of the official clients, then it shows up in Spot. Same with the other generated playlists.

**Comment by hb0nd** [comment]
+1 for this, the only reason i use Spotify instead of bandcamp etc is the algorithm's recommendations.

My reason for using Spot is that i'd rather use a proper libadwaita app. If the devs don't want to prominently feature the algorithm suggestions, maybe it could be hidden behind a toggle in settings by default?

**Comment by apposada** [comment]
+1 for this too. I think having a way of accessing Discover Weekly would be a great addition; maybe something like a section right together with 'starred tracks' and 'now playing' ?

**Comment by DimitrisMilonopoulos** [comment]
I would agree, I use spot as it is a GTK app, but I use spotify as it gives me great music discoverability, otherwise there is no reason to not prefer strawberry + tidal, or apple music, or any other streaming service :/
